apps/spruce/src/utils/getTaskFromMainlineCommitsQuery.ts

```diff
@@ -23,7 +23,6 @@ export const getTaskFromMainlineCommitsQuery = (
   }
   const buildVariant = buildVariants[0];
   if (buildVariant === null || buildVariant === undefined) {
-    reportError(new Error("buildVariant is undefined")).warning();
```

Comment:
Is there any reason not to remove all of the `reportError` instances above? These cases are handled; I'm not sure what value we get from logging.